devdocs/process.md

```diff
@@ -214,11 +217,13 @@ The meaning of issue prefixes: `s.` status, `a.` aspect, `f.` feature, `t.` type
 * Open PR  
     * `s.ToReview`: Waiting for the review
     * `s.Ongoing` : The PR is being worked on.
```

Comment:
a trivial thing, but should we have this as "OnGoing" to match the standard of capitalizing both words?